packages/devtools_app/lib/src/screens/inspector_v2/inspector_tree_controller.dart

```diff
@@ -507,6 +517,9 @@ class InspectorTreeController extends DisposableController
       gac.inspector,
       gac.treeNodeSelection,
     );
+    if (node?.diagnostic?.groupIsHidden ?? false) {
+      node?.diagnostic?.hideableGroupLeader?.toggleHiddenGroup();
```

Comment:
This can be `node!.diagnostic!.hideableGroupLeader?.toggleHiddenGroup();` Or better yet, you can store `node?.diagnostic` in a local var before the if check and then it will be promoted to nonnullable

Reply:
done!